guides/user/pipeline/triggers/github/index.md

```diff
@@ -16,18 +16,18 @@ files as [artifacts](/reference/artifacts) into a pipeline.
 
 This functionality uses GitHub
 [Webhooks](https://developer.github.com/webhooks/) for delivering messages to
-Spinnaker, and must be configured to send messages to Spinnaker's event bus as
+Spinnaker and must be configured to send messages to Spinnaker's event bus as
 shown below.
 
 # Prerequisite configuration/setup
 
-If you (or your Spinnaker admin) have already configured Spinnaker to listen to
+If you (or your Spinnaker admin) already configured Spinnaker to listen to
 a GitHub webhooks from the repository you plan to publish commits to, you can
 skip this section.
 
 You need the following:
 
-* A GitHub repository either under your user, or in an organization or user's
+* A GitHub repository either under your user, in an organization, or user's
```

Comment:
```suggestion
* A GitHub repository, either under your user account or in an organization or user's
```

Reply:
I think we need the "or" here. I took another shot at this one.

Reply:
Okay that's fine!

Reply:
+1 @dorbin